src/commonMain/kotlin/software/momento/kotlin/sdk/responses/cache/list/ListFetchResponse.kt

```diff
@@ -0,0 +1,26 @@
+package software.momento.kotlin.sdk.responses.cache.list
+
+import software.momento.kotlin.sdk.exceptions.SdkException
+
+/** Response for a list fetch operation */
+public sealed interface ListFetchResponse {
+
+    /** A successful list fetch operation that found elements. */
+    public data class Hit(private val byteArrayValues: List<ByteArray>) : ListFetchResponse {
+
+        /** Retrieves the values as a list of byte arrays. */
+        public fun valueListByteArray(): List<ByteArray> = byteArrayValues
```

Comment:
In the other responses that return something, we're exposing the values as properties instead of getter functions. `GetResponse` contains an example. `valueListByteArray` can be a public val and `valueListString` can be a lazy initializer.

Reply:
Very cool